armis_sdk/core/armis_client.py

```diff
@@ -87,18 +87,38 @@ def __init__(
         except ValueError:
             self._default_backoff = 0
 
+    def _get_proxy_config(self):
```

Comment:
Please put all private methods after all public method (and sorted within each group)